README.md

```diff
@@ -1,8 +1,7 @@
 dynamic-graph
 =============
 
-[![Building Status](https://travis-ci.org/stack-of-tasks/dynamic-graph.svg?branch=master)](https://travis-ci.org/stack-of-tasks/dynamic-graph)
-[![Pipeline status](https://gepgitlab.laas.fr/stack-of-tasks/dynamic-graph/badges/master/pipeline.svg)](https://gepgitlab.laas.fr/stack-of-tasks/dynamic-graph/commits/master)
+[![Building status](https://gepgitlab.laas.fr/stack-of-tasks/dynamic-graph/badges/master/pipeline.svg)](https://gepgitlab.laas.fr/stack-of-tasks/dynamic-graph/commits/master)
```

Comment:
Changing `[Pipeline status]` to `[Building status]` will not change the text in the svg generated by gitlab, which will still show a `pipeline` label: https://gepgitlab.laas.fr/stack-of-tasks/dynamic-graph/badges/master/pipeline.svg

This `[Pipeline status]` is only about the `alt` attribute of the `<img>` HTML tag, that will usually appear when the image can't be loaded, in case of:
- network issue on loading the image
- text rendering of the HTML page
- web navigator for people who can't see well (eg. audiodescription)

Reply:
So I suggest not to change this `Pipeline` into a `Building`, as this is the standard Markdown line provided by Gitlab:  https://gepgitlab.laas.fr/stack-of-tasks/dynamic-graph/-/settings/ci_cd (General pipelines >  Pipeline status)

Reply:
Done